packages/wrangler/src/index.tsx

```diff
@@ -1306,6 +1306,11 @@ function createCLIParser(argv: string[]) {
           type: "string",
           requiresArg: true,
         })
+        .option("do-not-build", {
```

Comment:
Thoughts on `no-build`? It's a little more concise

Reply:
This sounds like a much better name to me, personally

Reply:
done